### PR TITLE
Fix field names in json for CRD

### PR DESCRIPTION
**Pull request by anandswaminathan**
Noticed that some of the json field for certain attributes are different that intended. Fixing them here.

### CODE REVIEW

File: pkg/controller/flinkapplication/flink_state_machine.go [hunk 1]
```diff
@@ -129,6 +126,8 @@ func (s *FlinkStateMachine) Handle(ctx context.Context, application *v1alpha1.Fl
 
 	// Update k8s object
 	if updateApplication {
+		now := v1.NewTime(s.clock.Now())
```
Comment (mwylde) [inline pkg/controller/flinkapplication/flink_state_machine.go]: This change seems unrelated?

Comment (anandswaminathan) [reply pkg/controller/flinkapplication/flink_state_machine.go]: We are currently not updating LastUpdatedAt when JobStatus/ClusterStatus is modified. 

This seems more logical, to set `LastUpdatedAt` before we make the update call. Happy to open in a different PR/
